### PR TITLE
Re{factor/structure} the UI package and event administration activity

### PR DESCRIPTION
**Pull request by Fnux**
#145 reopened against master since #143 is already merged into master.

I user merges since I moved quite a lot of things and rebasing would be awful. I'd like to merge during this week.

### CODE REVIEW

**Comment by Fnux** [comment]
The history is quite ugly with all those merges. I will rebuild it (and squash everything into a single merge) once everything is ready.

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -24,7 +24,9 @@
 
         <activity
                 android:name=".ui.SplashActivity"
-                android:theme="@style/AppTheme.Launcher">
+                android:theme="@style/AppTheme.Launcher" />
+
+        <activity android:name=".ui.event.selection.EventPickingActivity">
```
Comment (Luminilion) [inline app/src/main/AndroidManifest.xml:31]: Does SplashActivity stay useful if it is not the Launcher anymore ?

File: app/src/main/java/ch/epfl/sweng/eventmanager/ui/event/interaction/EventAdministrationActivity.java [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+    @Inject
+    ViewModelFactory factory;
+
+    private EventInteractionModel model;
+    private int eventID;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        AndroidInjection.inject(this);
```
Comment (Luminilion) [inline app/src/main/java/ch/epfl/sweng/eventmanager/ui/event/interaction/EventAdministrationActivity.java:31]: Simple curiosity question here, is AndroidInjection the one used by Dagger ?

File: app/src/main/java/ch/epfl/sweng/eventmanager/ui/event/interaction/fragments/EventUserManagementFragment.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+import android.widget.Switch;
+import android.widget.TextView;
+
+import ch.epfl.sweng.eventmanager.R;
+import ch.epfl.sweng.eventmanager.ui.event.interaction.fragments.AbstractShowcaseFragment;
+
+/**
+ * A simple {@link Fragment} subclass.
+ * Activities that contain this fragment must implement the
+ * to handle interaction events.
```
Comment (Luminilion) [inline app/src/main/java/ch/epfl/sweng/eventmanager/ui/event/interaction/fragments/EventUserManagementFragment.java]: Maybe uncomplete JavaDoc no ?
`must implement the to handle`

File: app/src/main/res/layout/activity_event_administration.xml [hunk 4]
```diff
@@ -1,9 +1,35 @@
 <?xml version="1.0" encoding="utf-8"?>
-<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
+<!-- Use DrawerLayout as root container for activity -->
```
Comment (Luminilion) [inline app/src/main/res/layout/activity_event_administration.xml:2]: Simple curiosity again, how is it possible that the activity layout is a DrawerLayout ?

**Review by Luminilion — DISMISSED** [review]

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (Fnux) [reply app/src/main/AndroidManifest.xml:31]: I have no idea, it is code I got from master. Did I forgot something?

File: app/src/main/res/layout/activity_event_administration.xml (hunk 4 above)
Comment (Fnux) [reply app/src/main/res/layout/activity_event_administration.xml:2]: Why not? This construct is similar to `EventShowcaseActivity`.

File: app/src/main/java/ch/epfl/sweng/eventmanager/ui/event/interaction/EventAdministrationActivity.java (hunk 2 above)
Comment (Fnux) [reply app/src/main/java/ch/epfl/sweng/eventmanager/ui/event/interaction/EventAdministrationActivity.java:31]: What do you mean by "the one used by dagger" ?

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (Luminilion) [reply app/src/main/AndroidManifest.xml:31]: The code on Master is :
```
      <activity
           android:name=".ui.SplashActivity"
           android:theme="@style/AppTheme.Launcher">
            <intent-filter>
                <action android:name="android.intent.action.MAIN"/>
                <category android:name="android.intent.category.LAUNCHER"/>
            </intent-filter>
        </activity>
```

And EventPickingActivity is under meta

File: app/src/main/java/ch/epfl/sweng/eventmanager/ui/event/interaction/EventAdministrationActivity.java (hunk 2 above)
Comment (Luminilion) [reply app/src/main/java/ch/epfl/sweng/eventmanager/ui/event/interaction/EventAdministrationActivity.java:31]: More specifically is AndroidInjection used with Dagger ?

File: app/src/main/res/layout/activity_event_administration.xml (hunk 4 above)
Comment (Luminilion) [reply app/src/main/res/layout/activity_event_administration.xml:2]: Yes you're right sorry !

**Review by zbirobin — APPROVED** [review]

**Pull request merged**